wgsl/index.bs

```diff
@@ -9627,6 +9665,40 @@ Each is described in detail in subsequent sections.
       [=group_count_y=] - 1, [=group_count_z=] - 1).
 </table>
 
+##### `subgroup_invocation_id` ##### {#subgroup-invocation-id-builtin-value}
+
+<table class='data'>
+  <tr><td style="width:10%">Name
+      <td><dfn noexport dfn-for="built-in values">subgroup_invocation_id</dfn>
+  <tr><td style="width:10%">Stage
+      <td>[=compute shader stage|compute=] or [=fragment shader stage|fragment=]
+  <tr><td style="width:10%">Type
+      <td>u32
+  <tr><td style="width:10%">Direction
+      <td>Input
+  <tr><td style="width:10%">Description
+      <td>
+      The current invocation's [=subgroup invocation ID=].
+
+      The ID is within the range [0, [=built-in values/subgroup_size=] - 1].
+</table>
+
+##### `subgroup_size` ##### {#subgroup-size-builtin-value}
+
+<table class='data'>
+  <tr><td style="width:10%">Name
+      <td><dfn noexport dfn-for="built-in values">subgroup_size</dfn>
+  <tr><td style="width:10%">Stage
+      <td>[=compute shader stage|compute=] or [=fragment shader stage|fragment=]
+  <tr><td style="width:10%">Type
+      <td>u32
+  <tr><td style="width:10%">Direction
+      <td>Input
+  <tr><td style="width:10%">Description
+      <td>
+      The [=subgroup size=] of current invocation's subgroup.
```

Comment:
Should we mention that this falls between the `subgroupMinSize` and `subgroupMaxSize` values on `GPUAdapterInfo`?

Reply:
I put that in the definition of subgroup size and linked to the API spec.